cmd/bucket-replication.go

```diff
@@ -1068,7 +1068,7 @@ func (ri ReplicateObjectInfo) replicateObject(ctx context.Context, objectAPI Obj
 		VersionSuspended: versionSuspended,
 	})
 	if err != nil {
-		if !isErrObjectNotFound(err) {
+		if !isErrVersionNotFound(err) {
```

Comment:
```suggestion
		if !isErrVersionNotFound(err) &&  !isErrObjectNotFound(err) {
```
Couldn't the object have been deleted? I don't see much bad in checking both.